pipeline_steps/common.groovy

```diff
@@ -56,6 +56,15 @@ def install_ansible_roles(){
   """
 }
 
+def get_influx_credentials(){
+  return [
+    string(
```

Comment:
This can go directly in the withEnv call, unless you see this being used in multiple places?